build-parent/pom.xml

```diff
@@ -146,6 +146,10 @@
         <version.helm.plugin>6.14.3</version.helm.plugin>
         <version.helm.min>3.12.0</version.helm.min>
         <version.snakeyaml>2.0</version.snakeyaml>
+
```

Comment:
C (Critical): these do not belong here.  This is an open-source project, so we don't push anything into an internal repository nor reference an internal repository in its code base.

Please close this PR and remove the branch.

Reply:
I think I misunderstood the intention here.
I got it now, We need to create CI profile in the Video processing not aissemble root. closing.